### PR TITLE
Add reek config file support

### PR DESCRIPTION
**Pull request by zewelor**
Now its possible to use reek config file while running via pronto.

### CODE REVIEW

**Comment by mmozuras** [comment]
@zewelor could you also update the README? (example: https://github.com/mmozuras/pronto-rubocop) 😄

File: README.md [hunk 1]
```diff
@@ -6,3 +6,7 @@
 [![Dependency Status](https://gemnasium.com/mmozuras/pronto-reek.png)](https://gemnasium.com/mmozuras/pronto-reek)
 
 Pronto runner for [Reek](https://github.com/troessner/reek), code smell detector for Ruby. [What is Pronto?](https://github.com/mmozuras/pronto)
+
+## Configuration
+
+Configuring Reek via config.reek, or any file ending with .reek, will work just fine with pronto-reek.
```
Comment (mmozuras) [inline README.md]: @zewelor could you also link `config.reek` to https://github.com/troessner/reek#configuration-file? 😄

**Comment by zewelor** [comment]
@mmozuras Added link and rebased.

**Comment by mmozuras** [comment]
@zewelor thanks 🙇!

**Pull request merged**